libraries/standard/http/test/README.md

```diff
@@ -0,0 +1,17 @@
+# Instructions for running the tests:
+
+1. In your Linux environment using WSL, mingw, a Linux OS, etc. open a terminal.
+1. To get all the required installation scripts, clone this repo: https://github.com/dan4thewin/mock4thewin.
+1. Go to the repo directory: `cd \{mock4thewin repo}`
+1. Run this command to install: `sudo ./install /usr/local`
```

Comment:
Instead of installing it locally from the repository, can we host the build framework in this repository? I'm saying this from a long-term perspective of maintaining the Makefile

Reply:
When you say host, do you mean like setting up a CI system? That would be nice to setup.

Reply:
I am not setting up the test infrastructure right now. This is simply to get started and have tests. We can do this later.

Reply:
Leaving this out here for future reference: https://developer.github.com/apps/quickstart-guides/creating-ci-tests-with-the-checks-api/